plugins/__init__.py

```diff
@@ -395,8 +399,8 @@ def is_owner(user: discord.User):
     :param user: discord.User, discord.Member or a str representing the user's ID.
     :raises: TypeError: user is wrong type.
     """
-    if isinstance(user, discord.User):
-        user = user.id
+    if user is not None and user.bot is not True:
```

Comment:
this check was in place to verify that the passed argument was a subclass of discord.User or a string. it's not supposed to check for bot accounts. it seems like you interpreted this as the same problem as with private channels though.

i believe the previous code should still work for this purpose, so i guess attempt applying it (although this is unnecessary to test).

```suggestion
    if isinstance(user, discord.User):
```

Reply:
Resolved with https://github.com/pckv/pcbot/pull/18/commits/875fd0538f24b2b8958953fc671ae743d306d4b0